### DISCUSSION TITLE
what is device_id cookie?

### DISCUSSION THREAD

**Discussion by joaopa24** [community/community#121357] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

https://github.com/privacy/cookies

In the cookies part, would the device_id be the cookie to check if the device is trustworthy? Does this device change over time alone? I lost my account and the recovery methods, I'm trying to use device verification, but it's not enabled, the documentation says that this cookie expires in 1 year, but I logged into my account at the end of last year and I have proof of have at least logged in in July to upload a project. what do I do? support does not respond.

**Comment by ABDE-LKADER** [comment]
The device_id cookie can change over time due to factors like browser updates. If you're struggling with account recovery and support isn't responding, keep trying and provide evidence of recent activity to prove ownership.

**Reply by joaopa24** [reply]
unfortunately, even though I have complete proof that I am the owner of the account, if I lose the methods I cannot recover, they say it is a recovery policy.

**Comment by niladri-1** [comment]
Imagine a device_id cookie is like a little tag that a website puts on your computer or phone when you visit it. This tag helps the website remember your device when you come back later, kind of like how a sticker with your name helps people remember who you are at a party.
